staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -252,3 +347,94 @@ func isEmptyUnstructuredObject(in runtime.Object) bool {
 	}
 	return true
 }
+
+// validateConvertedObject checks that ObjectMeta fields match, with the exception of
+// labels and annotations.
```

Comment:
It is particularly noteworthy that apiVersion is not being compared here. Maybe focus on what the validation is achieving instead of which fields are excluded? (the full list of excluded fields is rather long)

Reply:
Oops, feel free to mark this as resolved.